mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -36,10 +36,6 @@ func resourceBigtableInstance() *schema.Resource {
 			},
 		},
 
-		// ----------------------------------------------------------------------
```

Comment:
Note:
We are moving ahead with removing this now that `deletion_protection` is available.